### ISSUE TITLE
DOI: 10.1162/089976698300017746

### ISSUE THREAD

**Issue by refraction-ray** [quclub/Paper-reading#54]
* [Link](https://www.mitpressjournals.org/doi/10.1162/089976698300017746)
* Title: Natural Gradient Works Efficiently in Learning
* Keywords (optional):  
* Authors (optional): 
* Reason (optional): NG is the same as SR in VMC, and can somehow be approximated by Adam
* Summary (optional): 
* Ideas (optional):

**Comment by refraction-ray** [comment]
## Relevant works on natural gradient

* [Revisiting natural gradient for deep networks](https://arxiv.org/pdf/1301.3584.pdf) *relation with Hessian free methods*
* [New insights and perspectives on the natural gradient method](https://arxiv.org/pdf/1412.1193.pdf) *relation between Fisher matrix and Hessian*
* [Accelerating Natural Gradient with Higher-Order Invariance](https://arxiv.org/pdf/1803.01273.pdf)
* [Optimization Methods for Large-Scale Machine Learning](https://arxiv.org/pdf/1606.04838.pdf) *A great review on optimizers that especially suitable for DL set up, see 6.4 for natural gradient analysis*
* [Limitations of the Empirical Fisher Approximation](https://arxiv.org/pdf/1905.12558.pdf) *empirical Fish may be not connected to real Fisher*
* [Optimizing Neural Networks with Kronecker-factored Approximate Curvature](http://proceedings.mlr.press/v37/martens15.pdf) *KFAC, an efficient way to approximate natural gradient*
* [Pathological spectra of the Fisher information metric
and its variants in deep neural networks](https://arxiv.org/pdf/1910.05992.pdf) *singularity on Fisher matrix, also applied to SR method*
* [Noisy Natural Gradient as Variational Inference](https://arxiv.org/pdf/1712.02390.pdf) *relation between NG and its role in VI*

**Comment by refraction-ray** [comment]
## Other resources on natural gradient
* [Post on Fisher information](https://wiseodd.github.io/techblog/2018/03/11/fisher-information/) and [post on natural gradient](https://wiseodd.github.io/techblog/2018/03/14/natural-gradient/)
* [Post on NG and its implementation in TensorFlow](https://squadrick.github.io/journal/natural-gradients-in-tensorflow.html)
* [Post on KFAC](https://towardsdatascience.com/introducing-k-fac-and-its-application-for-large-scale-deep-learning-4e3f9b443414)

**Comment by refraction-ray** [comment]
Comment: I am not independently realizing that stochastic reconfiguration method in VMC is the same thing as natural gradient descent. The idea is from this paper: PHYSICAL REVIEW X 8, 011006 (2018). This is a physics paper on RBM wavefunction VMC, however, the authors mentioned in this work:
> stochastic reconfiguration optimization, which is equivalent to the natural gradient descent

Great observation! And it is impressive that some optimization based on imaginary time dynamics and Schrodinger equations can be fully reduced to general optimization setups. But I am not sure who is the first one that realized this connection.

Idea: If I have time, I will think about imaginary time evolution in TNS settings, which is also used for optimization ground state energy. And I will definitely check whether such "physicsy" optimization scheme can also be reduced to some general well-established optimization scheme in optimization field.

**Update:** this relation is in particular be clarified in Append B of arXiv:1909.02487. But the authors mentioned in this work that either community doesn't realize it before. No, the above PRX is the evidence.
Another paper on this: [On the geometry of learning neural quantum states](https://arxiv.org/pdf/1910.11163.pdf)
